assets/css/staff.css

```diff
@@ -14,7 +14,7 @@ p {
     flex-wrap: wrap;
 }
 
-.staff-card {
+.staff-card{
```

Comment:
nitpick: We probably don't need to make this change 🙂

Reply:
Oops I think that was a typo, thanks for catching it!